editor/components/block-list/breadcrumb.js

```diff
@@ -34,27 +25,59 @@ function stopPropagation( event ) {
  * @param {string}   props.uid             UID of block.
  * @param {string}   props.rootUID         UID of block's root.
  * @param {Function} props.selectRootBlock Callback to select root block.
- *
- * @return {WPElement} Breadcrumb element.
  */
-function BlockBreadcrumb( { uid, rootUID, selectRootBlock } ) {
-	return (
-		<NavigableToolbar className="editor-block-breadcrumb">
-			<Toolbar>
-				{ rootUID && (
-					<Tooltip text={ __( 'Select parent block' ) }>
-						<Button
-							onClick={ selectRootBlock }
-							onFocus={ stopPropagation }
-						>
-							<Dashicon icon="arrow-left" uid={ uid } />
-						</Button>
-					</Tooltip>
-				) }
-				<BlockTitle uid={ uid } />
-			</Toolbar>
-		</NavigableToolbar>
-	);
+export class BlockBreadcrumb extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			isFocused: false,
+		};
+		this.onFocus = this.onFocus.bind( this );
+		this.onBlur = this.onBlur.bind( this );
+	}
+
+	onFocus( event ) {
+		this.setState( {
```

Comment:
Why was this needed?

Reply:
In the breadcrumb component at the moment no because it's not shown at all when the block is selected, but I didn't want to write the implementation depending on something not in the component itself. If tomorrow we decide to show this breadcrumb even on selected blocks, this will become necessary.